README.md

```diff
@@ -242,6 +228,14 @@ Run the web app container using:
 
     docker run --net='container:max-im2txt' -it webapp
 
+##### Using the Docker Hub Image
+
+You can also deploy the web app with the latest docker image available on DockerHub by running:
+
+    docker run --net='container:max-im2txt' -it codait/max-image-caption-generator-web-app
```

Comment:
Same comment applies here, changing `max-im2txt` to `max-image-caption-generator`